README.txt

```diff
@@ -1 +1,3 @@
 Temporary repo for Git / GitHub training
+
+This is for 2016/09/28
```

Comment:
This does look like today's date.  Did you mean to use international format, or should this be US date format?